config/locales/en.yml

```diff
@@ -211,6 +211,7 @@ en:
         already_added: "You have already added this managing agent"
       merged: "That organisation has already been merged. Select a different organisation."
     not_answered: "You must answer %{question}"
+    range: "%{question} should be between %{min} and %{max}"
```

Comment:
we've got validations.numeric.within_range that might be usable instead in this case (it has slightly different phrasing though), your call :)

Reply:
Eh, I'm not going to quibble over should and must - let's get rid of this.